dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -336,16 +336,27 @@
 			label = "TRNG";
 		};
 
-		usbd2: usbd@402e0200 {
+		usbd1: usbd@402e0000 {
 			compatible = "nxp,kinetis-usbd";
-			reg = <0x402E0200 0x1000>;
+			reg = <0x402E0000 0x200>;
 			interrupts = <113 1>;
 			interrupt-names = "usb_otg";
 			num-bidir-endpoints = <8>;
 			maximum-speed = "full-speed";
 			status = "disabled";
 			label = "USBD_1";
 		};
+
+		usbd2: usbd@402e0200 {
+			compatible = "nxp,kinetis-usbd";
```

Comment:
compatible should be `"nxp,mcux-ehci-usbd"` but there is no binding file `nxp,mcux-ehci-usbd.yaml`

Reply:
Hi @jfischer-phytec-iot , I just reuse the nxp,kinetis-usbd.yaml. Is it neccessary to add new file like nxp,mcux-ehci-usbd.yaml? If it is neccessary, I think it is better to do it in another PR.

Reply:
@jfischer-phytec-iot is right this should be a different compatible because it is different hardware IP, however I think the compatible should be `nxp,imx-usbd`. Yes you will need to add a new yaml for it, and yes that can be done in another PR.